NineChronicles.Headless.Executable/Program.cs

```diff
@@ -125,7 +125,9 @@ static void ConfigureSentryOptions(SentryOptions o)
             [Option(Description = "The AWS region for AWS CloudWatch (e.g., us-east-1, ap-northeast-2).")]
             string awsRegion = null,
             [Option(Description = "Run as an authorized miner, which mines only blocks that should be authorized.")]
-            bool authorizedMiner = false
+            bool authorizedMiner = false,
+            [Option(Description = "The lifetime of each Tx, It's unit is minute.  60 (m) by default.")]
```

Comment:
What you intended is below code, doesn't it?

```suggestion
            [Option(Description = "The lifetime of each Tx, its unit is minute.  60 (m) by default.")]
```

And how about using the full word? (e.g. `tx` and `transaction`)

```suggestion
            [Option(Description = "The lifetime of each transaction, which uses minute as its unit.  60 (m) by default.")]
```

Reply:
fix on https://github.com/planetarium/NineChronicles.Headless/pull/258/commits/ded53676b59171f558faf35a8d7d1b340eee3df4